<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Number Of Frames:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="image_label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>500</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="data_set_label">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Select Data Set:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="data_set_combo_box">
        <item>
         <property name="text">
          <string>Ball</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>YellowBall</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Player</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Erase</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hand</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="frames_line_edit">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="open_image_button">
        <property name="text">
         <string>Open Image</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="roi_button">
        <property name="text">
         <string>Select ROI</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="tracking_button">
        <property name="text">
         <string>Start Tracking</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Stop Tracking</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHistogram">
    <property name="title">
     <string>Histogram</string>
    </property>
    <addaction name="actionIntensity_Histogram"/>
    <addaction name="actionColor_Histogram"/>
   </widget>
   <addaction name="menuHistogram"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionIntensity_Histogram">
   <property name="text">
    <string>Intensity Histogram</string>
   </property>
  </action>
  <action name="actionNormalize_Histogram">
   <property name="text">
    <string>Normalize Histogram</string>
   </property>
  </action>
  <action name="actionColor_Histogram">
   <property name="text">
    <string>Color Histogram</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
